package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -560,10 +560,15 @@ def _read_next_timestep(self):
 
         triclinic = len(f.readline().split()) == 9  # ITEM BOX BOUNDS
         if triclinic:
-            xlo, xhi, xy = map(float, f.readline().split())
-            ylo, yhi, xz = map(float, f.readline().split())
+            xlo_bound, xhi_bound, xy = map(float, f.readline().split())
```

Comment:
As far as I can tell the maths here is correct. I'm going to ping @orbeckst to possibly verify? 

You should also leave a comment as to why this is the necessary and where someone can find the information (https://docs.lammps.org/Howto_triclinic.html) on this if they need to. 👍

Reply:
The math in https://github.com/MDAnalysis/mdanalysis/pull/3403/files#r700652018 looks correct. 

Definitely include a link to https://docs.lammps.org/Howto_triclinic.html in the docs and as a comment — it's an excellent resource.